.travis.yml

```diff
@@ -260,7 +267,6 @@ _helm_deploy: &helm_deploy
         helm upgrade $release_name_acs helm/alfresco-content-services \
         --wait \
         --timeout 10m0s \
-        --atomic \
```

Comment:
I know you removed this for testing but did you intend to leave it out of forget to add back?

Reply:
removed on purpose